repository-module/src/main/java/com/gmail/supersonicleader/app/repository/impl/GeneralRepositoryImpl.java

```diff
@@ -0,0 +1,8 @@
+package com.gmail.supersonicleader.app.repository.impl;
+
+import com.gmail.supersonicleader.app.repository.GeneralRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
```

Comment:
No need to put an annotation on abstract class where we cannot create the object